### PR TITLE
Add email to volunteers about the SR2022 Competition

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

**Review by antoinepetty — APPROVED** [review]

File: SR2022/2022-01-20-competition-volunteer-signup.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Please note that, unlike previous years, we are not able to reimburse travel or
+accommodation at the competition, though we hope to provide ways for volunteers
+who are interested to group together to save costs.
+
+    [Volunteer at the Competition](https://forms.gle/B5FACojyniQrnM2N8)
+
+We're also looking for people to help organise the competition. Even if you only
+have a small amount of time available, every contribution helps make the event
+better for the competitors. Please contact competiton-team@studentrobotics.org
+if you'd like to help.
```
Comment (RealOrangeOne) [inline SR2022/2022-01-20-competition-volunteer-signup.md:26]:
```suggestion
if you'd like to help, or come along to a competition meeting.
```

**Review by RealOrangeOne — APPROVED** [review]

File: SR2022/2022-01-20-competition-volunteer-signup.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2022/2022-01-20-competition-volunteer-signup.md]: https://github.com/srobo/volunteer-emails/pull/49/commits/55f77c6fdeb05fc73696e08471f59913e88ead84

File: SR2022/2022-01-20-competition-volunteer-signup.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+to: All Volunteers
+subject: Help run the SR2022 Competition event (dates finalised)
```
Comment (PeterJCLaw) [inline SR2022/2022-01-20-competition-volunteer-signup.md:3]: @Tyler-Ward could you suggest a subject line which would have been more compelling?

Comment (Tyler-Ward) [reply SR2022/2022-01-20-competition-volunteer-signup.md:3]: some quick ideas.
Please let us know if you can help at this years competition
Signup to help at SR2022

**Pull request merged**